ann_benchmarks/algorithms/faiss.py

```diff
@@ -55,11 +53,17 @@ def fit(self, X):
         index = faiss.IndexIVFFlat(self.quantizer, X.shape[1], self._n_list, faiss.METRIC_L2)
         index.train(X)
         index.add(X)
-        index.nprobe = self._n_probe
         self._index = index
 
+    def set_query_arguments(self, n_probe):
+        self._n_probe = n_probe
```

Comment:
Could anyone of you please correct this typo? It has to be `self.nprobe = n_probe`, as seen above. Otherwise we don't set the probe count for IVF.